src/components/VAudioTrack/VWaveform.vue

```diff
@@ -468,9 +467,19 @@ export default defineComponent({
       }
     }
 
+    const handleSpacebar = () => {
+      emit('toggle-playback')
+    }
+
+    /**
+     * @param {KeyboardEvent} event
+     */
     const willBeHandled = (event) =>
-      ['ArrowLeft', 'ArrowRight', 'Home', 'End'].includes(event.key)
+      ['ArrowLeft', 'ArrowRight', 'Home', 'End', ' '].includes(event.key)
```

Comment:
I looked it up, @zackkrida  :D It used to be `Spacebar`, in the older versions of browsers

Reply:
I've update the PR to include using contants for the keycodes instead of magic strings. Hopefully that clears up the confusion without having to add explanatory comments.